### DISCUSSION TITLE
Menu Options Breaks after loading an suspended window

### DISCUSSION THREAD

**Discussion by conmaster2112** [Mojang/minecraft-editor#160] [Bugs]
Every time i minimize and reopen minecraft, then it adds new bugged menu option to bar.
here is image after one re-minimize
![image](https://github.com/Mojang/minecraft-editor/assets/64863757/d144c8d8-401f-49a0-9ee2-6eb379f0c17b)
after second re-minimize
![image](https://github.com/Mojang/minecraft-editor/assets/64863757/30dbcd32-b5f7-425e-a5b2-0241240e1e24)
....
![image](https://github.com/Mojang/minecraft-editor/assets/64863757/517e7be1-bdd8-4f10-9d2c-4a6eec62a392)

Preview build version: 1.20.60.23
Frequency (eg. Always, Sometimes, Rarely): Always
Local single-player, peer-to-peer, dedicated server (and how many people): Local Single Player
Graphics card (eg. DX11 or DX12): DX12+

**Comment by m-brisch** [comment]
This should now be fixed with the menu upgrades in [v0.5.10](https://github.com/Mojang/minecraft-editor/discussions/168)!
